koza/model/config/source_config.py

```diff
@@ -186,6 +186,9 @@ class SourceConfig:
     json_path: List[Union[StrictStr, StrictInt]] = None
     transform_code: str = None
     transform_mode: TransformMode = TransformMode.flat
+    global_table: str = None
```

Comment:
if we set this as a string does this break if we go the !include route later? or is the intention not to use include here because it should be treated as a file path?

Reply:
It seemed so straightforward to use it as a file path that I just stopped there... but I could imagine that for a small local table it would be nice to just put it in the config rather than a separate file (which I assume would get accessed the same way in the code as the !include), should I see if I can get that to work?

Reply:
I think the metadata yaml works this way, but I think either way is fine